src/Storages/System/StorageSystemStoragePolicies.cpp

```diff
@@ -24,9 +24,10 @@ StorageSystemStoragePolicies::StorageSystemStoragePolicies(const std::string & n
              {"volume_name", std::make_shared<DataTypeString>()},
              {"volume_priority", std::make_shared<DataTypeUInt64>()},
              {"disks", std::make_shared<DataTypeArray>(std::make_shared<DataTypeString>())},
-             {"max_data_part_size", std::make_shared<DataTypeUInt64>()},
```

Comment:
Why we lost it?

Reply:
I guess if we want it here then we should move `max_data_part_size` to `IVolume`. @alesapin do we want it in `IVolume` or not?

Reply:
Yes this is sad. @NanoBjorn

Reply:
I've taken `max_data_part_size` back. @alesapin please check.